### DISCUSSION TITLE
Add a new bulk action item

### DISCUSSION THREAD

**Discussion by johanboekhoven** [aminalaee/sqladmin#483] [Q&A] [answered]
I want to add another bulk action to the action dropdown in the listview, adding a menu item is easy enough, but how do I set up the rest of the code to process the selected items with my custom action.
For example I want to change the state of the selected row by setting an archived flag to true.

Thanks for any pointers!

**Comment by aminalaee** [comment]
Hey,
It was once tried in https://github.com/aminalaee/sqladmin/pull/273
Right now it's not possible, I will work on this and hopefully it will be in version 0.11.0 soon.

**Reply by johanboekhoven** [reply]
Thanks for the quick answer @aminalaee, appreciate that you want to take a look! It would be a great addition to the UI.

**Reply by aminalaee** [reply, marked as answer]
Thanks to @murrple-1 This is now merged: https://github.com/aminalaee/sqladmin/issues/484
